### ISSUE TITLE
The specified module could not be found with v1.7.0-pre

### ISSUE THREAD

**Issue by grische** [Esri/palladio#141]
I downloaded v1.7.0-pre and installed it into the houdini17.5 folder inside my documents:

```
[pld] [2019-11-12 22:04:03] [error] failed to load library: error while loading library 'C:\Users\grische\Documents\houdini17.5\packages\palladio\prtlib\com.esri.prt.unreal.dll': The specified module could not be found.
[pld] [2019-11-12 22:04:03] [error] failed to load library: error while loading library 'C:\Users\grische\Documents\houdini17.5\packages\palladio\prtlib\DatasmithSDK.dll': The specified module could not be found.
```

But the files are present
```
PS C:\Users\grische> dir C:\Users\grische\Documents\houdini17.5\packages\palladio\prtlib\


    Directory: C:\Users\grische\Documents\houdini17.5\packages\palladio\prtlib


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------       2019-10-20   2:45 PM         317440 com.esri.prt.adaptors.dll
------       2019-10-20   2:45 PM        1146368 com.esri.prt.alembic.dll
------       2019-10-20   2:45 PM       27289600 com.esri.prt.codecs.dll
------       2019-10-20   2:45 PM         374784 com.esri.prt.unreal.dll
------       2019-10-20   2:45 PM       14277632 DatasmithSDK.dll
------       2019-10-20   2:45 PM        5751296 FreeImage317.dll
------       2019-10-20   2:45 PM        1634816 LibRaw.dll
------       2019-10-20   2:46 PM          81920 palladio_codec.dll
------       2019-10-20   2:45 PM         958976 VueExport.dll
```

As I am using `Houdini FX Version 17.5.425`, is this a duplicate of #139 ?

**Comment by mistafunk** [comment]
do the pldAssign/pldGenerate SOPs show up at all in Houdini? if not, here's a workaround: delete these 4 dlls from the prtlib directory: com.esri.prt.unreal.dll, DatasmithSDK.dll, FreeImage317.dll, LibRaw.dll

background: your system is likely lacking some dependencies required by the PRT unreal encoder. which is no problem as palladio does not use this encoder anyways for Houdini. proper fix will be to tell the palladio build system to exclude the unreal stuff (and other unused extensions while we're at it).

**Comment by grische** [comment]
The SOPs do show up, but the error message goes away after removing the mentioned libraries.

**Comment by mistafunk** [comment]
merged, released in 1.7.0-beta1
